geonames/geonames.php

```diff
@@ -158,7 +160,7 @@ function geonames_addon_admin_post($a,$post) {
 
 
 
-function geonames_addon_admin(&$a,&$s) {
+function geonames_addon_admin(App $a, $s) {
```

Comment:
The `&` operator enables this function to add new settings to the admin. If you remove it the Geonames settings won't show up.